### PR TITLE
raftstore/apply: Temporarily skip delete all in range

### PR DESCRIPTION
**Pull request by MyonKeminta**
Please refer to #3034 
This PR skips the second step of delete range (`delete_all_in_range`) temporarily.
However in order to make this change, some tests about delete range have to be bypassed, which is somewhat dangerous. Please make sure we do really need this before merging this.
Related pull request in TiDB: https://github.com/pingcap/tidb/pull/6512

### CODE REVIEW

**Comment by MyonKeminta** [comment]
@zhangjinpeng1987 PTAL

**Comment by zhangjinpeng1987** [comment]
If use_delete_range is true, don't skip the cleanup process.

**Comment by siddontang** [comment]
LGTM

**Comment by MyonKeminta** [comment]
@BusyJay

File: tests/raftstore/cluster.rs [hunk 1]
```diff
@@ -594,20 +595,23 @@ impl<T: Simulator> Cluster<T> {
...
 
-    fn get_impl(&mut self, key: &[u8], read_quorum: bool) -> Option<Vec<u8>> {
-        let mut resp = self.request(
-            key,
-            vec![new_get_cmd(key)],
-            read_quorum,
-            Duration::from_secs(5),
-        );
+    fn get_impl(&mut self, cf: &str, key: &[u8], read_quorum: bool) -> Option<Vec<u8>> {
+        let req = match cf {
```
Comment (zhangjinpeng1987) [inline tests/raftstore/cluster.rs]: We can reuse new_get_cf_cmd when cf is default.

File: tests/raftstore/cluster.rs [hunk 2]
```diff
@@ -109,6 +109,7 @@ impl<T: Simulator> Cluster<T> {
         sim: Arc<RwLock<T>>,
         pd_client: Arc<TestPdClient>,
     ) -> Cluster<T> {
+        // TODO: In the future, maybe it's better to test both true and false
```
Comment (huachaohuang) [inline tests/raftstore/cluster.rs]: What's this?

File: src/server/service/kv.rs [hunk 3]
```diff
@@ -585,6 +585,7 @@ impl<T: RaftStoreRouter + 'static> tikvpb_grpc::Tikv for Service<T> {
         ctx.spawn(future);
     }
 
+    // WARNING: Currently this API may last some dirty keys in TiKV. Be careful using this API.
```
Comment (huachaohuang) [inline src/server/service/kv.rs]: s/last/leave

**Pull request merged**